pkgs/development/python-modules/online-judge-tools/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, colorama
+, fetchFromGitHub
+, online-judge-api-client
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "online-judge-tools";
+  version = "11.5.1";
```

Comment:
this should probably be a separate commit, but this PR is ancient, and don't want to cause yet-another-turnaround